spacemesh0.1.md

```diff
@@ -14,23 +14,23 @@ This guide walks you though and describes the features and the supported functio
 
 Here are some scenarios that are not yet supported in the Spacemesh 0.1 Testnet release:
 
-1. Join the Testnet from a Node and a CLI wallet built directly from source code.
-2. Get Smesh coins from a tap.
+1. Join the Testnet from a Node and a CLI wallet built directly from source code. 
```

Comment:
Is it really not supported? What happens if I compile the Node and tries to connect to the netwrok?

Reply:
I wish we'll be able to support this but I from my understanding more work is involved and in the last meet it was requested to delay it to 0.x. It is beyond just joining the network - this is about being able to see post, execute txs w the cli wallets and see awards in the cli wallet - in other words, a reasonable user experience.